pkg/tuned/tuned.go

```diff
@@ -370,11 +380,17 @@ func (c *Controller) tunedRun() {
 	scanner := bufio.NewScanner(cmdReader)
 	go func() {
 		for scanner.Scan() {
-			fmt.Printf("%s\n", scanner.Text())
+			l := scanner.Text()
+			if c.daemon.reloading {
+				c.daemon.reloading = !(strings.Index(l, "static tuning from profile") >= 0 && strings.Index(l, "applied") >= 0)
+				c.daemon.reloaded = !c.daemon.reloading
+			}
+			fmt.Printf("%s\n", l)
 		}
 	}()
 
 	err = c.tunedCmd.Start()
+	c.daemon.reloading = true
```

Comment:
I'm confused by the lines above, I'm afraid that there might be a race condition, but I have 0 prior knowledge about Goroutines, so take this comment with caution:
* `c.tunedCmd.Start()` unblocks `scanner.Text()`, and the goroutine checks `c.daemon.reloading`
* the main routine sets `	c.daemon.reloading = true`

```suggestion
	c.daemon.reloading = true
	err = c.tunedCmd.Start()
```

Reply:
This is a great catch, thank you!  Will fix.